input/pages/packaging.md

```diff
@@ -56,7 +56,7 @@ NOTE: It is recommended that each resource in the transaction is a [conditional
       "request" :{
         "method": "POST",
         "url": "Library",
-        "ifNotExist": "url=http://example.org/Library/SomeLibrary&version=0.1.0"
+        "ifNoneExist": "url=http://example.org/Library/SomeLibrary&version=0.1.0"
```

Comment:
The ticket and the spec indicate the header is `If-None-Exist`, does that need to be the name of the element in the request JSON body here?

Reply:
Oh I see, this is the bundle property, got it